wrappers/services.go

```diff
@@ -550,6 +558,13 @@ WantedBy={{.ServicesTarget}}
 		restartCond = snap.RestartOnFailure.String()
 	}
 
+	// start with score -900-1 because snapd itself has OOMScoreAdjust=-900
```

Comment:
probably easier to just make the comment be about the whole block:

//  use score -900+vitalityRank, where vitalityRank starts at 1 and considering snapd itself has OOMScoreAdjust=-900